capella_model_explorer/app.py

```diff
@@ -159,6 +159,13 @@ def rendered_report(template_id: str, model_element_uuid: str = "") -> t.Any:
             cls="prose svg-display dark:prose-invert",
         )
     )
+    # set max-age to 1 year if the model is versioned
+    max_age = 31536000 if state.model.info.resources["\x00"].rev_hash else 0
```

Comment:
Comments should never explain _what_ happens. That's already visible in the code.

I do think this case could get an explanation on _why_ this happens though.

```suggestion
    # Non-Git model versions cannot be distinguished reliably in the cache
    max_age = 31536000 if state.model.info.resources["\x00"].rev_hash else 0
```